<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="send_move_bt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Move</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_edit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Move (1-9)</string>
    </property>
   </widget>
   <widget class="QLabel" name="board_lbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>441</width>
      <height>291</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>[GAME BOARD]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="error_lbl">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>[Error Label]</string>
    </property>
   </widget>
   <widget class="QLabel" name="turn_lbl">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>[Turn Label]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chat_input_box">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type message here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_btn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="chat_output_box">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>281</width>
      <height>471</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
